client/i18n/locales/english/intro.json

```diff
@@ -2858,6 +2858,14 @@
           "Test what you've learned in this quiz on JavaScript Fundamentals Quiz."
         ]
       },
+      "plic": { "title": "182", "intro": [] },
+      "vjmm": { "title": "183", "intro": [] },
+      "lab-book-organizer": {
+        "title": "Build a Book Organizer",
+        "intro": [
+          "In this lab, you will build a book organizer using higher order functions in JavaScript."
+        ]
+      },
```

Comment:
This can go below the library manager. Not that it really matters, but we've been trying to keep them in order.